frontend/src/component/Activity/ActivityList.styl

```diff
@@ -20,7 +20,7 @@ previewHeight = 400px
 
     &__overflowOverlay
       background-image:
-        linear-gradient(to top, rgba(0, 0, 0, 0.25), transparent),
+        linear-gradient(to top, rgba(0, 0, 0, 0.1), transparent),
```

Comment:
`linear-gradient(to top, rgba(0, 0, 0, 0.1), transparent)` should be `linear-gradient(to top, rgba(1, 1, 1, 1), transparent)`
So the gradient will fade to the white

Reply:
`rgba(1, 1, 1, 1)` it's black, to fade to white it's `rgba(255, 255, 255, 1)`. I can update this, the initial change was just "make it a little closer to the gray of the card".